src/main/java/pl/edu/agh/firecell/model/Material.java

```diff
@@ -7,25 +7,28 @@
 public enum Material {
     // TODO:
     // consider putting them into constant
-    AIR(0, 0, 0, FLUID, 900),
-    WOOD(3, 250, 0, SOLID, 550);
+    AIR(0, 0, 0, FLUID, 900, 0),
+    WOOD(3, 250, 0, SOLID, 550, 30);
 
     private final double ignitionTemperature;
     private final double autoIgnitionTemperature;
     private final double fireSpreadSpeed;
     private final MatterState matterState;
     private final double burningTemperature;
+    private final int smokeCoe;
```

Comment:
```suggestion
    private final int smokeCoefficient;
```
No need to shorten the names in such small data-class.